A32NX/html_ui/Pages/A32NX_Core/FMGC/A32NX_FlightPhaseManager.js

```diff
@@ -131,15 +131,26 @@ class A32NX_FlightPhaseManager {
 class A32NX_FlightPhase_PreFlight {
     constructor() {
         this.nextFmgcFlightPhase = FmgcFlightPhases.TAKEOFF;
+        this.takeoffConfirmation = new NXLogic_ConfirmNode(.2);
     }
 
     init(_fmc) {
         _fmc.climbTransitionGroundAltitude = null;
     }
 
-    check(_fmc) {
+    check(_deltaTime, _fmc) {
         // Simplane.getAltitudeAboveGround() > 1500; is used to skip flight phase in case ac reloaded midair
-        return Simplane.getEngineThrottleMode(0) >= ThrottleMode.FLEX_MCT || Simplane.getEngineThrottleMode(1) >= ThrottleMode.FLEX_MCT || Simplane.getAltitudeAboveGround() > 1500;
+        return this.takeoffConfirmation.write((
+            (
+                Simplane.getEngineThrottleMode(0) >= ThrottleMode.FLEX_MCT ||
+                Simplane.getEngineThrottleMode(1) >= ThrottleMode.FLEX_MCT
+            ) && (
+                SimVar.GetSimVarValue("ENG N1 RPM:1", "percent") > .75 ||
+                SimVar.GetSimVarValue("ENG N1 RPM:2", "percent") > .75
+            ))
+        || Simplane.getAltitudeAboveGround() > 1500,
+        _deltaTime
+        );
```

Comment:
Personally I find it more readable like this (prettier-style):

```suggestion
        return this.takeoffConfirmation.write(
            (
                (
                    Simplane.getEngineThrottleMode(0) >= ThrottleMode.FLEX_MCT ||
                    Simplane.getEngineThrottleMode(1) >= ThrottleMode.FLEX_MCT
                ) && (
                    SimVar.GetSimVarValue("ENG N1 RPM:1", "percent") > .75 ||
                    SimVar.GetSimVarValue("ENG N1 RPM:2", "percent") > .75
                )
            ) || Simplane.getAltitudeAboveGround() > 1500,
            _deltaTime
        );
```